apps/site/lib/site_web/templates/event/index.html.eex

```diff
@@ -7,25 +7,53 @@
         month: @month,
         year: @year %>
 
-      <%# Add calendar toggle, setting the active option based on if "calendar" query_param is true %>
+      <% params = Enum.map(@conn.query_params, fn({key, val}) -> {String.to_existing_atom(key), val} end) %>
+
+      <div class="m-events-hub__nav--navigation-toggle">
+        <% index_params = Keyword.delete(params, :calendar) %>
+        <%= link to: event_path(@conn, :index, index_params), role: "button", tabindex: "0" do %>
+          <i class="fa fa-list-ul m-nav-toggle-icon <%= if !assigns[:calendar_view], do: "active-view" %>"></i>
+        <% end %>
```

Comment:
Each button is available to screen readers, but is just called "button".  Give the buttons a descriptive label (I think using `aria-label`?)